src/taipy/core/_repository/_v2/_abstract_repository.py

```diff
@@ -23,7 +23,7 @@
 Entity = TypeVar("Entity")
 
 
-class _AbstractRepository(Generic[ModelType, Entity]):  # type: ignore
+class _AbstractRepository(Generic[ModelType, Entity]):
```

Comment:
Yeah !!! we removed a type ignore. We are more often prone to add new ones...

Reply:
This one I had added as well :smile:, but now it's gone.

Reply:
👍🏻